provider/cmd/pulumi-resource-random/schema.json

```diff
@@ -66,8 +32,8 @@
                     "description": "The generated id presented in base64, using the URL-friendly character set: case-sensitive letters, digits and the characters `_` and `-`.\n"
                 },
                 "byteLength": {
-                    "type": "integer",
```

Comment:
integer vs number looks suspect. Need to dig in if we can 'understand' things are actually an integer only.